app/src/main/java/org/groundplatform/android/ui/util/FormatConversion.kt

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2025 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.groundplatform.android.ui.util
+
+import android.content.res.Resources
+import android.icu.text.MeasureFormat
+import android.icu.util.LocaleData
+import android.icu.util.Measure
+import android.icu.util.MeasureUnit
+import android.icu.util.ULocale
+import android.os.Build
+import java.util.Locale
+import org.groundplatform.android.R
+
+fun formatDistance(resources: Resources, distanceInMeters: Double): String {
```

Comment:
The filename seems to imply this is a class which represents a conversion, rather than a series of functions. Can these be moved into a new or existing Ext.kt file?

Reply:
The idea was to use this new file "FormatConversion" as a single place where all the functions which are responsible of format conversion will get placed here

Reply:
Great. Want to turn this into an extension function, `Double.formatDistance(Resources): String`? I'd suggest keeping this file more narrowly scoped, eg `DistanceFormatExt.kt`